### DISCUSSION TITLE
Support `loading` prop for `MultiSelect` component

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#2204] [Feature requests]
Currently, we are using `MultiSelect` for loading async data and we are able to display a loading spinner with `rightContent` prop and show that no results were found with `nothingFound` prop: https://mantine.dev/core/multi-select/#searchable

The UX team wants to display a "Loading..." text instead of "Nothing found" text while API results are still pending. Can this be supported? I didn't find a way.

**Comment by rtivital** [comment]
Hi @mcmxcdev, you can change nothing fount message with `nothingFound` prop. For example, when your request is not finished yet, you can set `nothingFound="Leading data"`, then when data is ready to be used you can change the message to be `nothingFount="Nothing found"`.

**Reply by mcmxcdev** [reply]
There are probably more valid cases where a `loading` prop would be helpful, but I will use it conditionally as you say ;)
